content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -0,0 +1,121 @@
+---
+title: Menjalankan Multipel Instans dari Aplikasimu
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Penyekalaan aplikasi menggunakan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+       <h3>Penyekalaan sebuah Aplikasi</h3>
+
+            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment yang dibuat itu hanya satu Pod yang menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
+
+            <p><b>Penyekalaan</b> dapat dicapai dengan mengubah nilai replicas dalam Deployment</p>
```

Comment:
```suggestion
            <p><b>Penyekalaan</b> dapat dicapai dengan mengubah jumlah replika dalam Deployment</p>
```

Reply:
this one arguable because of this reverence to `.spec.replicas` field on the Deployment.

Reply:
Yes, I know that there's a `replicas` field for Deployment `spec`. When I read the original text I think it more as a general sentence, since it's not specifically mentioning that replicas is a field there (it's ambiguous whether it means replicas in plain English tech term or `replicas` field). Taken for granted updating the `replicas` field value means changing the number of replicas for a Deployment.

It's just my personal view btw...

Reply:
More discussions here https://kubernetes.slack.com/archives/CJ1LUCUHM/p1600235247012200